### PR TITLE
fix(components): Adding aria-label to close buttons in components

### PR DESCRIPTION
**Pull request by VadymBezpalko**
## Description
Adding aria-label to close buttons in components

## Storybook

https://feature-close-buttons-aria--613a8e945a5665003a05113b.chromatic.com

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [x] Assign pull request with the correct issue

### CODE REVIEW

**Review by MateuszRomek — APPROVED** [review]

File: packages/react-components/src/components/ActionMenu/ActionMenu.tsx [hunk 1]
```diff
@@ -77,6 +77,10 @@ export interface ActionMenuProps {
    * https://floating-ui.com/docs/usefloating#strategy
    */
   floatingStrategy?: Strategy;
+  /**
+   * Test id passed to the wrapper element
+   */
+  ['data-testid']?: string;
```
Comment (sgraczyk) [inline packages/react-components/src/components/ActionMenu/ActionMenu.tsx:83]: @VadymBezpalko Have you considered an extendable Props interface for all components, so some core fields would be part of every single component? `['data-testid']` could be a part of them. 

Such approach would prevent you from necessity to add them from time to time when needed.

**Review by sgraczyk — APPROVED** [review]

File: packages/react-components/src/components/ActionMenu/ActionMenu.tsx [hunk 2]
```diff
@@ -251,7 +256,8 @@ export const ActionMenu: React.FC<ActionMenuProps> = ({
   return (
     <>
       <div
-        data-testid="action-menu-trigger-button"
```
Comment (MrVorgoth) [inline packages/react-components/src/components/ActionMenu/ActionMenu.tsx:254]: If it was passed before, shouldn't it stay and be overwritten by custom data-test id if passed?

File: packages/react-components/src/components/ActionMenu/ActionMenu.tsx (hunk 1 above)
Comment (VadymBezpalko) [reply packages/react-components/src/components/ActionMenu/ActionMenu.tsx:83]: Great idea 👍

**Review by MrVorgoth — APPROVED** [review]

**Review by vladko-uxds — APPROVED** [review]

File: packages/react-components/src/components/ActionMenu/ActionMenu.tsx (hunk 2 above)
Comment (MateuszRomek) [reply packages/react-components/src/components/ActionMenu/ActionMenu.tsx:254]: Its set as default value in props

**Pull request merged**